demo/meeting/back-end-resources/back-end.ts

```diff
@@ -0,0 +1,196 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import dynamodb = require('@aws-cdk/aws-dynamodb');
+import lambda = require('@aws-cdk/aws-lambda');
+import { Construct } from '@aws-cdk/core';
+import cdk = require('@aws-cdk/core');
+import apigateway = require('@aws-cdk/aws-apigateway'); 
+import iam = require('@aws-cdk/aws-iam')
+import loggroup = require('@aws-cdk/aws-logs');
+
+export class BackEnd extends Construct {
+    constructor(parent: Construct, name: string) {
+        super(parent, name);
+
+        const meetingsTable = new dynamodb.Table(this, 'meetings', {
+            partitionKey: {
+              name: 'Title',
+              type: dynamodb.AttributeType.STRING
+            },
+            tableName: 'meetings',
+            removalPolicy: cdk.RemovalPolicy.DESTROY,
+            timeToLiveAttribute: 'TTL',
+            billingMode: dynamodb.BillingMode.PAY_PER_REQUEST,            
+          });
+
+        const attendeeTable = new dynamodb.Table(this, 'attendees', {
+          partitionKey: {
+            name: 'AttendeeId',
+            type: dynamodb.AttributeType.STRING
+          },
+          tableName: 'attendees',
+          removalPolicy: cdk.RemovalPolicy.DESTROY,
+          billingMode: dynamodb.BillingMode.PAY_PER_REQUEST
+        });
+
+        const lambdaChimeRole = new iam.Role(this, 'LambdaChimeRole', {
+          assumedBy: new iam.ServicePrincipal('lambda.amazonaws.com'),
+        });
+    
+        lambdaChimeRole.addToPolicy(new iam.PolicyStatement({
+          resources: ['*'],
+          actions: ['chime:CreateMeeting',
+                    'chime:CreateMeeting',
+                    'chime:CreateMeeting',
+                    'chime:DeleteMeeting',
+                    'chime:GetMeeting',
+                    'chime:ListMeetings',
+                    'chime:BatchCreateAttendee',
+                    'chime:CreateAttendee',
+                    'chime:DeleteAttendee',
+                    'chime:GetAttendee',
+                    'chime:ListAttendees',
```

Comment:
This can likely be tuned better.  I just took them from the original: https://github.com/aws/amazon-chime-sdk-component-library-react/blob/091d371167145bd95aaac68848b7abcf9dce3499/demo/meeting/serverless/template.yaml#L33

Reply:
Maybe we can just remove the duplicate CreateMeetings then

Reply:
Done.